dockers/docker-orchagent/ipinip_ping.py

```diff
@@ -0,0 +1,233 @@
+#! /usr/bin/env python3
+"""
+Adds neighbor to kernel for undeliverable IPinIP packets
```

Comment:
Suggest renaming this to` tunnel_packet_handler.py` so we can extend this to Vxlan in future